common/api/ui-components.api.md

```diff
@@ -3474,10 +3481,16 @@ export interface PropertyGridCategory {
 // @internal (undocumented)
 export class PropertyGridCommons {
     // (undocumented)
-    static assignRecordClickHandlers(records: PropertyRecord[], onPropertyLinkClick?: (property: PropertyRecord, text: string) => void): void;
+    static assignRecordClickHandlers(records: PropertyRecord[], onPropertyLinkClick: (text: string) => void): void;
     // (undocumented)
     static getCurrentOrientation(width: number, preferredOrientation?: Orientation, isOrientationFixed?: boolean, horizontalOrientationMinWidth?: number): Orientation;
-    }
+    static getLinks: (value: string) => Array<{
+        start: number;
+        end: number;
+    }>;
+    // @beta

```

Comment:
This doesn't seem right - we should not be adding `@beta` functions to `@internal` classes. Does it need to be `@beta`?